metricsbp/prometheus.go

```diff
@@ -0,0 +1,24 @@
+package metricsbp
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var activeRequests = promauto.NewGauge(prometheus.GaugeOpts{
+	Name: "active_requests",
```

Comment:
Is this specific to a protocol? Typically for this metric we would prefix it with the protocol.

This can be done with the `Namespace`. Something like this:

```
prometheus.GaugeOpts{
	Namespace: "http",
	Name: "active_requests",
...
}
```

Reply:
sounds like the best thing to do is move this to the per transport server middleware. will make that change.